src/test/run-fail/meta-revision-bad.rs

```diff
@@ -13,10 +13,18 @@
 
 // revisions: foo bar
 // should-fail
-//[foo] error-pattern:bar
-//[bar] error-pattern:foo
+// [foo] error-pattern:bar
+// [bar] error-pattern:foo
```

Comment:
Probably best to manually fix this for now. I intend to change the default to not touch comments in the next release of rustfmt, so long term (well, medium term) it shouldn't be a problem